addons/victoria-metrics-cluster/values.yaml

```diff
@@ -0,0 +1,36 @@
+# Default values for victoria-metrics-cluster.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+clusterVersionOverride: ""
+
+terminationPolicy: "Delete"
+
+## @param vminsert configs
+##
+vminsert:
+  replicas: 1
+  resources: {}
+
+## @param vmselect configs
+##
+vmselect:
+  replicas: 1
+  resources: {}
+
+## @param vmstorage configs
+##
+vmstorage:
```

Comment:
add an extra field,  such that users can specify their storage class name.
e.g. refer to `storageClassName` in apecloud-mysql.